### ISSUE TITLE
Fatal Exception: java.lang.NullPointerException

### ISSUE THREAD

**Issue by SergiOnGit** [zo0r/react-native-push-notification#1641]
# Bug
App crashes and closing. Firebase crashlitycs catches this error "Attempt to invoke virtual method 'double java.lang.Double.doubleValue()' on a null object reference"

## Environment info

`react-native info` output:

```bash
 System:
    OS: Windows 10 10.0.18363
    CPU: (8) x64 Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz
    Memory: 4.93 GB / 15.89 GB
  Binaries:
    Node: 10.19.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.21.1 - C:\Users\Warlock\AppData\Roaming\npm\yarn.CMD
    npm: 6.13.4 - C:\Program Files\nodejs\npm.CMD
    Watchman: Not Found
  SDKs:
    Android SDK:
      API Levels: 23, 25, 27, 28, 29
      Build Tools: 19.1.0, 20.0.0, 21.1.2, 22.0.1, 23.0.1, 23.0.2, 23.0.3, 24.0.0, 24.0.1, 24.0.2, 24.0.3, 25.0.0, 25.0.1, 25.0.2, 25.0.3, 26.0.0, 26.0.1, 26.0.2, 26.0.3, 27.0.0, 27.0.1, 27.0.2, 27.0.3, 28.0.0, 28.0.1, 28.0.2, 28.0.3, 29.0.0, 29.0.1, 29.0.2, 29.0.3, 30.0.0, 30.0.1, 30.0.2
      System Images: android-28 | Google Play Intel x86 Atom, android-29 | Google APIs Intel x86 Atom, android-29 | Google Play Intel x86 Atom
      Android NDK: Not Found
  IDEs:
    Android Studio: Version  4.0.0.0 AI-193.6911.18.40.6626763
  Languages:
    Java: 1.8.0_231
    Python: 2.7.17
  npmPackages:
    @react-native-community/cli: Not Found
    react: 16.13.1 => 16.13.1
    react-native: 0.63.2 => 0.63.2
  npmGlobalPackages:
    *react-native*: Not Found
```

Library version: 5.1.0

## Reproducible sample code

Here is my code:

I am canceling notification and then register.

`PushNotification.cancelLocalNotifications({
            id: '090000',
        });`

```
PushNotification.localNotificationSchedule({
            id: '090000',
            message: 'test',
            date: morningDate,
            repeatType: 'day',
            allowWhileIdle: true,
            autoCancel: true,
            ignoreInForeground: true,
});
```

Doesn't crash anywhere, only some devices or maybe randomly cause it happens on every model and every android version.

**Comment by Dallas62** [comment]
Hi @SergiOnGit 
Can you provide the full stacktrace ?
Actually the mentioned method is not used in the project.
https://github.com/zo0r/react-native-push-notification/search?q=doubleValue&unscoped_q=doubleValue

**Comment by kabus202** [comment]
I'm doing the same, and i have exactly the same problem

**Comment by Dallas62** [comment]
Hi @kabus202 
Thanks for the stacktrace, I'm on it.

**Comment by Dallas62** [comment]
Can you try the version on master ?

**Comment by SergiOnGit** [comment]
Hello @Dallas62 

Thanks for fast response. Latest version on master is 5.1.0 and i already have it. So what do you mean?

**Comment by Dallas62** [comment]
If you are able to reproduce it, you can test:

```
npm install zo0r/react-native-push-notification#master
```
OR
```
yarn add zo0r/react-native-push-notification#master
```

**Comment by SergiOnGit** [comment]
I got this. Give me one day to test it.

**Comment by kabus202** [comment]
yes, i will test this also today (afternoon)

**Comment by SergiOnGit** [comment]
I have updated my app in production yesterday and firebase doesn't see any error in new version. So i think it is solved. @kabus202 What about you?

**Comment by kabus202** [comment]
hello guys, i don't see any errors, i think it's fixed @SergiOnGit 
@Dallas62 thanks for the quick fix, really appreciate it.

**Comment by Dallas62** [comment]
Hi,
Thanks for feedbacks!
I will release the fix at the end of the day. As a fix-version (5.1.1).
Regards

**Comment by SergiOnGit** [comment]
Thanks you @Dallas62
